sdk/digitaltwins/azure-digitaltwins-core/API design.md

```diff
@@ -280,7 +277,237 @@ When updating a relationship edge, the patch string follows the below format
 <details><summary><b>APIs</b></summary>
 
 ```java
-TODO:
+    /**
+     * Creates a relationship on a digital twin.
+     *
+     * @param digitalTwinId The Id of the source digital twin.
+     * @param relationshipId The Id of the relationship to be created.
+     * @param relationship The application/json relationship to be created.
+     * @return The application/json relationship created.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public String createRelationship(String digitalTwinId, String relationshipId, String relationship) { }
```

Comment:
We need to confirm from the SDK board if this needs to be a String -> autorest defines it as an Object, which is actually a map

Reply:
For context, the data returned is not a strongly typed object; it has some predefined service-related metadata, and additional user-defined properties.